serverless/lambda/src/main/java/uk/gov/di/services/ConfigurationService.java

```diff
@@ -39,6 +39,10 @@ public String getNotifyApiKey() {
         return System.getenv("NOTIFY_API_KEY");
     }
 
+    public long getEmailCodeExpiry() {
+        return Long.parseLong(System.getenv().getOrDefault("EMAIL_CODE_EXPIRY", "900"));
```

Comment:
are we sure we want a default here shouldn't it blow up if it is not present. Reason being if some looks at the repo they can work out the length of email code which is a possible security risk

Reply:
I could remove the default but the environment variables are currently set in this repo within the terraform anyway so it won't solve the problem of people being able to view it.

Reply:
lets chat with the team about this for the time lets leave it how the others are set